ExpressAPI/db.js

```diff
@@ -0,0 +1,26 @@
+const mysql = require('mysql');
+
+const db = mysql.createConnection({
+    host: process.env.DB_HOST,
```

Comment:
Do i get the env vars on my machine locally?, just not sure how to config this so it runs on my machine.

Reply:
Yes, you'll need to make a .env file with those values inside your ExpressAPI folder and add those values in as per your machine.

Reply:
thanks, that's all my comments